src/Sylius/Behat/Client/ApiClientInterface.php

```diff
@@ -35,5 +43,11 @@ public function getError(): string;
 
     public function isCreationSuccessful(): bool;
 
+    public function isUpdateSuccessful(): bool;
```

Comment:
I'm not sure, but this method name is not consistent with the one above 😄

Reply:
I believe `update` is also a noun, not only a verb